packages/amplify_auth_plugin_interface/lib/src/Exceptions/InvalidStateException.dart

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_auth_plugin_interface/amplify_auth_plugin_interface.dart';
+
+/// Thrown when an operation is not valid with the current state of Auth category 
+class InvalidStateException extends AuthException {
+  /// Named constructor
```

Comment:
Nit: Don't think this is a named constructor
```suggestion
  /// Default constructor
```

Reply:
You are right. This is what I get for copy and pasting code 🙂